test.js

```diff
@@ -8,8 +8,8 @@ var assert = require('chai').assert;
       assert.equal(gql`{ testQuery }`.kind, 'Document');
     });
 
-    it('returns the same object for the same query', () => {
```

Comment:
This was actually the intention - downstream tools use `===` to compare identical queries:

```js
gql`...` === gql`...`;
```

Why do we need to make this change?